### PR TITLE
[4.0] [a11y] Do not link intro images

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue https://github.com/joomla/cassiopeia/issues/121.

### Summary of Changes
In Blog layouts the intro-image is a link if the param "llink-title" is set. This is an accessibility issue, 
This PR removes the linked intro image.

### Testing Instructions
In a blog entry select an into-image for an article. 
Set the param "link-titles" in the menuItem to this blog. 

### Actual result BEFORE applying this Pull Request
The image is a link to the article, and the title is also a link to the article. 

### Expected result AFTER applying this Pull Request
The image is not a link. 

### Note
An alternative solution would be a new param "link_intro_image", but in this case linking the title must be excluded. 

### Documentation Changes Required
no

### CODE REVIEW

**Comment by brianteeman** [comment]
My 2c would be to have a link param with a default option of link_title and the second option of link_intro_image
This would mean you only have one link which satisfies the a11y rule on adjacent identical links.

**Comment by brianteeman** [comment]
But this is definitely a valid solution as well

File: layouts/joomla/content/intro_image.php [hunk 1]
```diff
@@ -18,19 +18,10 @@
...
-			</a>
-		<?php else : ?>
-			<img loading="lazy" src="<?php echo htmlspecialchars($images->image_intro, ENT_COMPAT, 'UTF-8'); ?>"
-				 alt="<?php echo htmlspecialchars($images->image_intro_alt, ENT_COMPAT, 'UTF-8'); ?>"
-				 itemprop="thumbnailUrl"
-			>
-		<?php endif; ?>
+		<img loading="lazy" src="<?php echo htmlspecialchars($images->image_intro, ENT_COMPAT, 'UTF-8'); ?>"
+				alt="<?php echo htmlspecialchars($images->image_intro_alt, ENT_COMPAT, 'UTF-8'); ?>"
+				itemprop="thumbnailUrl"
```
Comment (Quy) [inline layouts/joomla/content/intro_image.php]:
```suggestion
			alt="<?php echo htmlspecialchars($images->image_intro_alt, ENT_COMPAT, 'UTF-8'); ?>"
			itemprop="thumbnailUrl"
```

**Comment by chmst** [comment]
> 
> 
> My 2c would be to have a link param with a default option of link_title and the second option of link_intro_image
> This would mean you only have one link which satisfies the a11y rule on adjacent identical links.

Maybe a better solution.

**Comment by brianteeman** [comment]
There is another option if the aim is to make the image and the title both clickable and yet only a single link by borrowing some css from bootstrap
https://getbootstrap.com/docs/4.5/utilities/stretched-link/

Works quite well on my site

**Comment by adj9** [comment]
I have tested this item :white_check_mark: successfully on a6bb8a5241b39c28f6f58e4a96ae43aab5225805

By inserting the image here and applying the PR, it is not clickable
Schermata 2020-09-26 alle 12 42 55

<br /><br />Confirmed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30769">issues.joomla.org/tracker/joomla-cms/30769</a>.</sub>

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on a6bb8a5241b39c28f6f58e4a96ae43aab5225805<br /><br />The patch works as described but for me the image is in the wrong place! It is above the Article title and frankly not obviously part of it. I think it should be closer to the first paragraph like the full article image. It is styled as float-left but its container is full width. But I guess where the Intro image appears is a separate issue.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30769">issues.joomla.org/tracker/joomla-cms/30769</a>.</sub>

**Comment by brianteeman** [comment]
@chmst did you mean to close this?

**Comment by chmst** [comment]
Indeed we should re-think this. It tile is linked AND a readmore is added, then there are two links ... so re-opening this.

**Comment by chmst** [comment]
@ceford placing the image is indeed another issue and we are working on it.

**Pull request closed without merging**